mycroft/util/parse.py

```diff
@@ -164,6 +169,571 @@ def extractnumber_en(text):
     return val
 
 
+'''authors: spenrod and connorpenrod
```

Comment:
Move this into the function using docstrings, like what you see in the extractnumber() method.  We use Google style docstrings (see:  http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html)

You can leave the Authors: stuff off.  That's all tracked by git anyway :)  I'm going to be removing the various Author tags from files at some point and moving everything into a master Contributor list instead.